src/Microsoft.AspNetCore.SystemWebAdapters/HttpPostedFileBase.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+
+namespace System.Web;
+
+public abstract class HttpPostedFileBase
+{
+    public virtual string FileName => throw new NotImplementedException();
```

Comment:
@johnLwith for these kind of warnings/errors, suppress with an attribute, and use the justification of `Constants.ApiFromAspNet`